tests/Helpers/UtilTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * This file is part of the php-resque package.
+ *
+ * (c) Michael Haynes <mike@mjphaynes.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\Resque\Heplers\Utils;
+
+use Resque\Helpers\Util;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class UtilTest
+ *
+ * @author Romain DARY <romain.dary@eoko.fr>
+ */
+class UtilTest extends TestCase
+{
+    /**
+     * @param string $expected expected readable sizes
+     * @param integer $bytes size in bytes
```

Comment:
Please use the shorthand type documentations (`int`/`bool`) to be in line with Symfony's code style and the proposed [PSR-5 standard](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md).